go/cmd/dolt/commands/init.go

```diff
@@ -43,7 +43,7 @@ Running dolt init in an already initialized directory will fail.
 `,
 
 	Synopsis: []string{
-		//`[{{.LessThan}}options{{.GreaterThan}}] [{{.LessThan}}path{{.GreaterThan}}]`,
+		"",
```

Comment:
Should this be an empty slice?

Reply:
Making it an empty slice will cause help and markdown to omit the synopsis section. Probably okay for this simple command, but it's nice for consistency in the docs for all commands to have the same sections.